### PR TITLE
Lizzie's edits to contributing page

### PR DESCRIPTION
**Pull request by emcdevitt**
I walked through all the instructions and added some additional text and reorganized some steps in a way that made things a little bit clearer to me (e.g., I moved the "Only do this once" part about adding the upstream remote to the "Things to set up once" section.

### CODE REVIEW

File: docs/contributing.rst
Comment (snastase) [inline docs/contributing.rst]: Except for, e.g., the `contributing.rst` etc, which lives in `handbook/docs`

File: docs/contributing.rst
Comment (snastase) [inline docs/contributing.rst]: I'm not sure this strictly necessary in all cases; might want to soften the wording a bit, more like "If your forked repo (on GitHub) gets behind the original repo, do X"

File: docs/contributing.rst
Comment (snastase) [inline docs/contributing.rst]: "remind"

**Review by snastase — CHANGES_REQUESTED** [review]
Okay, I went through @emcdevitt's changes and everything looks good to me. I made a couple minor comments on wording. Will defer to @paulapbrooks for approval.

File: docs/contributing.rst
Comment (paulapbrooks) [reply docs/contributing.rst]: Good catch, Sam! Lizzie, can you change this in your code? If you push the new code to your branch you don't have to submit a new pull request.

**Comment by snastase** [comment]
It occurs to me that we may want to add a small section on deleting your feature branch after the PR gets accepted and your changes are incorporated into master. For example, here's the recommendations from step 9 at the [BrainIAK contributing page](https://brainiak.org/docs/contributing.html):

```
git checkout master
git pull --ff-only
git branch -d new-feature
git push --delete origin new-feature  # or use delete button in GitHub PR
```

File: docs/contributing.rst
Comment (paulapbrooks) [inline docs/contributing.rst]: @emcdevitt This will build, but for aesthetics, can you shorten the dashed lines here so that it aligns with the new title?

File: docs/contributing.rst
Comment (paulapbrooks) [reply docs/contributing.rst]: LOL yea, but I guess the thought it that people should only be making changes to something in the content_pages folder?...

Comment (paulapbrooks) [reply docs/contributing.rst]: We could soften the language and say something like "........" Actually, I got nothing. @emcdevitt, do you have any thoughts?

**Review by paulapbrooks — CHANGES_REQUESTED** [review]
Hey Lizzie! ( @emcdevitt) ...I'm still getting a hang of the workflow on git :P 
Can you please take a look at the comments here and make changes (or continue the conversation if things aren't resolved) before we merge your pull request? Thanks for all of your suggestions/contributions! :)

File: docs/contributing.rst
Comment (emcdevitt) [reply docs/contributing.rst]: @snastase @paulapbrooks I'll just add something noting the exception...

File: docs/contributing.rst
Comment (emcdevitt) [reply docs/contributing.rst]: I'll add a note about this...

File: docs/contributing.rst
Comment (emcdevitt) [reply docs/contributing.rst]: donesies!

**Comment by emcdevitt** [comment]
> It occurs to me that we may want to add a small section on deleting your feature branch after the PR gets accepted and your changes are incorporated into master. For example, here's the recommendations from step 9 at the [BrainIAK contributing page](https://brainiak.org/docs/contributing.html):
> 
> ```
> git checkout master
> git pull --ff-only
> git branch -d new-feature
> git push --delete origin new-feature  # or use delete button in GitHub PR
> ```

@snastase These instructions are already included in step #10.

**Comment by emcdevitt** [comment]
@snastase @paulapbrooks I just pushed the requested changes. Hope I did it right!

**Comment by snastase** [comment]
> > It occurs to me that we may want to add a small section on deleting your feature branch after the PR gets accepted and your changes are incorporated into master. For example, here's the recommendations from step 9 at the [BrainIAK contributing page](https://brainiak.org/docs/contributing.html):
> > ```
> > git checkout master
> > git pull --ff-only
> > git branch -d new-feature
> > git push --delete origin new-feature  # or use delete button in GitHub PR
> > ```
> 
> @snastase These instructions are already included in step #10.

Oh good—missed that!

File: docs/contributing.rst
Comment (emcdevitt) [reply docs/contributing.rst]: done!

File: docs/contributing.rst
Comment (snastase) [inline docs/contributing.rst:90]: We should specify "(on Mac)" somewhere here... Also, do we really want to create this `~/Desktop/princeton_handbook/handbook/`... nested structure? For example something like `~/Desktop/handbook` would be sufficient, or (my example) `~/Work/brainhack-princeton/handbook` or just `~/Work/handbook`

Comment (snastase) [reply docs/contributing.rst:90]: I added "(e.g., on Mac)"... If nobody has strong feelings about the directory structure, we can ignore this comment since it doesn't really matter!

**Review by snastase — APPROVED** [review]

**Comment by snastase** [comment]
Maybe somebody should have a final read-through, but otherwise I'm ready to accept!

**Pull request merged**
